wiki/Rules/en.md

```diff
@@ -1,77 +1,81 @@
 
-1.  No spamming or flooding.
-2.  Evading silences will immediately escalate to a restriction if you are found doing it, and you will be found.
-    1.  Talking about silences (your own previous or someone else's) in public chats is forbidden and will be punished with a silence.
-3.  Public discussion of NSFW content or anything explicitly sexual is not permitted as per community rule \#5. Private messages are not covered by this rule.
-4.  All links to external content (images, threads, videos, etc) must not contain photosensitive content (very fast flashing strobes, etc).
-5.  Impersonation is explicitly forbidden, especially of staff.
-6.  Do not advertise multiplayer games outside of \#lobby.
-7.  Discussing cheating, hacking, or any other form of client abuse or foul play is explicitly forbidden.
-    1.  Mentioning or talking about banned players is forbidden as well.
-8.  In language-specific channels, speak the dominant language if you are participating in discussion. Where the language for a channel is not explicitly stated, it is always English.
-    1.  Moderators might talk English in language-specific channels as they see need for it to handle the situation appropriately.
+1. **No topic in Off-Topic has value.** They are put simply, trash. If your threads keep getting moved to Off-Topic, they are being judged to be unworthy of serious discussion. Take the hint.
+2. **Excessive stupidity is bad.** Being endearingly dumb is fine, and falls into the modus operandi of shitposting. Genuine stupidity is never fine. If you are unable to distinguish the difference, you're already breaking the rule.
+3. **Birthday threads are allowed only in Off-Topic, and nowhere else.** All birthday threads must be accompanied with no less than three (3) images of dancing anime girls or they will be *ruthlessly* thrown under a bus.
 
-Client rules
-------------
+What happens if I break the rules?
+---------------
 
-1.  Manipulation or alteration of the client such as score hacking, macroing, affecting the timescale on which the client operates, auto-aim, and anything that generally removes the human element from play is considered cheating. If you have to ask if something is considered cheating, and you have to use an external program to achieve that effect, it probably is.
-2.  Altering gameplay elements through any method BESIDES beatmap skins/client skins is considered as possible cheating and will be dealt with harshly.
+If you do find yourself on the wrong end of our community rules, bad things will happen.
 
-------------
+These bad things may include (*but are not limited to*):
 
-Cheaters are permanently banned and restricted upon the first offense and are forced to wait 6 months MINIMUM before they may appeal the ban. Just don't do it. Bans are rarely, if ever, made in error.
+* The removal of your ability to chat in-game for a period of time (known as a silence)
```

Comment:
it's not a huge deal so long as it's consistent in the singular articles themselves